releases/v19.2.0.md

```diff
@@ -0,0 +1,131 @@
+[**Streaming with JDBC**](../v19.2/build-a-java-app-with-cockroachdb.html#retrieve-large-data-sets-in-chunks-using-cursors) | CockroachDB now supports Postgres wire-protocol cursors for implicit and explicit transactions executed to completion. The Java JDBC driver can use this protocol to stream queries with large result sets, providing much faster performance than result paginating with `LIMIT ... OFFSET`.
+[**Transaction Latency Graphs**](../v19.2/admin-ui-sql-dashboard.html) | The **SQL Dashboard** in the Admin UI now provides timeseries graphs of p90 and p99 transaction latencies to compliment the per-statement metrics on the [**Statements** page](../v19.2/admin-ui-statements-page.html).
+**`CREATE TABLE AS` Improvements** | When using the `CREATE TABLE AS` statement to create a new table from a selection query, you can now [specify the primary key](../v19.2/create-table-as.html#specify-a-primary-key) and [define column families](../v19.2/create-table-as.html#define-column-families) for the new table. The latter is especially helpful for [modifying the primary key of tables for geo-partitioning](../v19.2/create-table-as.html#specify-a-primary-key-for-partitioning).
+[**Adding `NOT NULL` Constraint**](../v19.2/alter-column.html#set-not-null-constraint) | It's now possible to add the `NOT NULL` constraint to an existing column, in addition to the previously ability to set this constraint during table creation.
+**Local Testing Improvements** | CockroachDB v19.2 includes several usability improvements to running CockroachDB locally for SQL testing and app development. First, the new [`cockroach start-single-node`](../v19.2/cockroach-start-single-node.html) CLI command starts a single-node cluster with replication disabled without the need for any extra commands. Alongside this change, using `cockroach start` without the `--join` flag is now deprecated and will be removed in a future release.<br><br>Next, the [`cockroach demo`](../v19.2/cockroach-demo.html) command, which starts a temporary, in-memory CockroachDB cluster with immediate access to an interactive SQL shell, now comes pre-loaded with the `movr` dataset and a temporary enterprise license that expires after an hour. This means users can try out [enterprise features](../v19.2/enterprise-licensing.html) quickly before requesting a license for use in a persistent cluster.<br><br>Also, the `--nodes` flag now lets you specify the number of in-memory nodes to create for the demo, with the new `--demo-locality` flag setting locality information for each node. Alternatively, the `--geo-partitioned-replicas` flag lets you start a 9-node demo cluster with the [Geo-Partitioned Replicas](../v19.2/topology-geo-partitioned-replicas.html) data topology applied automatically to the `movr` database.<br><br>Finally, the `--with-load` command lets you run `movr` as an active workload against the in-memory cluster.
```

Comment:
> Alongside this change, using `cockroach start` without the `--join` flag is now deprecated and will be removed in a future release.

This particular sentence is not really about "local testing". it's rather a general operational change.